pkg/roachprod/install/cluster_synced.go

```diff
@@ -2439,6 +2440,24 @@ func (c *SyncedCluster) SSH(ctx context.Context, l *logger.Logger, sshArgs, args
 	return syscall.Exec(sshPath, allArgs, os.Environ())
 }
 
+var sshVersion3Internal struct {
+	value bool
+	once  sync.Once
+}
+
+// sshVersion3 returns true if ssh uses an SSL library at major version 3.
+func sshVersion3() bool {
+	sshVersion3Internal.once.Do(func() {
+		cmd := exec.Command("ssh", "-V")
+		out, err := cmd.CombinedOutput()
+		if err != nil {
+			panic(fmt.Sprintf("error running ssh -V: %v", err))
+		}
+		sshVersion3Internal.value = strings.Contains(string(out), "SSL 3.")
```

Comment:
We could try parsing to version to future this against version 4. But eh, I suppose that is next decades problem.

Reply:
Plus if history is any guide, next version might be 5. Or 9? :)